src/main/java/com/davidson/quote/QuoteController.java

```diff
@@ -71,6 +77,7 @@ public ResponseEntity updateQuote(@PathVariable("quoteid") Long id, @RequestBody
      * @return quote that has the id
      */
     @RequestMapping(value = "/quotes/{quoteid}/validate", method = RequestMethod.PUT)
+    @ApiOperation("Allows you to validate status send for quote")

```

Comment:
allows to validate a quote

Reply:
Pareil pour le commentaire de la javadoc : Enleve le endpoint et met : allows to validate quote